src/StockServiceManager.php

```diff
@@ -32,12 +50,13 @@ public function addService(StockServiceInterface $stock_service) {
    * {@inheritdoc}
    */
   public function getService(PurchasableEntityInterface $entity) {
-    $config = \Drupal::config('commerce_stock.service_manager');
```

Comment:
I'd keep this $config var, but set it to $this->configFactory->get('commerce_stock.service_manager'); so the $default_service_id and $entity_service_id don't have to duplicate code and can just be $config->get(key);